api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,328 @@
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/muted Set muted 
+     * @apiName televisionSetMuted
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {boolean} muted true if sound is muted
+     */
+    setMuted: function(req, res, next){
+        gladys.television.setMuted(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+     /**
+     * @api {post} /television/volume Set Volume 
+     * @apiName televisionSetVolume
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {integer} volume Volume level
+     */
+    setVolume: function(req, res, next){
+        gladys.television.setVolume(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/stop Stop 
+     * @apiName televisionStop
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    stop: function(req, res, next){
+        gladys.television.stop(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volumeUp Volume Up 
+     * @apiName televisionVolumeUp
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeUp: function(req, res, next){
+        gladys.television.volumeUp(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volumeDown Volume Down 
+     * @apiName televisionVolumeDown
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeDown: function(req, res, next){
+        gladys.television.volumeDown(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/pressKey Press Key 
+     * @apiName televisionPressKey
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    pressKey: function(req, res, next){
+        gladys.television.pressKey(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/getSources Get Sources
+     * @apiName televisionGetSources
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {array} All television source
+     */
+    getSources: function(req, res, next){
+        gladys.television.getSources(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/openSources Open Sources
```

Comment:
This is not really REST, we should find another way of doing that :)  

What is this command supposed to do ?

Reply:
This command is supposed to "switch tv to a new source"
Example : 
On TV you have 4 sources, HDMI1, HDMI2, AV, VGA.
If you want to switch to HDMI1 from the AV source, you have to send openSources to your TV with the HDMI1 source on parameter.

Are you agree ?